lib/matplotlib/figure.py

```diff
@@ -1220,12 +1220,18 @@ def colorbar(
                 fig.colorbar(cm.ScalarMappable(norm=norm, cmap=cmap), ax=ax)
 
         cax : `~matplotlib.axes.Axes`, optional
-            Axes into which the colorbar will be drawn.
+            Axes into which the colorbar will be drawn.  If `None`, then a new
+            Axes is created and the space for it will be stolen from the Axes(s)
+            specified in *ax*.
 
         ax : `~.axes.Axes` or iterable or `numpy.ndarray` of Axes, optional
-            One or more parent axes from which space for a new colorbar axes
-            will be stolen, if *cax* is None.  This has no effect if *cax* is
-            set.
+            If `None`, defaults to the axes that contains the mappable used to
+            create the colorbar.
+
+            If *cax* is None, specifies the one or more parent axes from which
+            space for a new colorbar axes will be stolen.
+
+            This has no effect if *cax* is set.
```

Comment:
```suggestion
            The one or more parent Axes from which space for a new colorbar Axes
            will be stolen. This parameter is only used if *cax* is not set.

            Defaults to the Axes that contains the mappable used to create the
            colorbar.
```
- IMHO "what is this" should come before "default"
- I also think that "what is this" should come before "when does this apply" (but you can also keep two if senctence for the *cax* None/not-None cases.
- `If None, defaults to ...` -> `Defaults to ...` The value `None` is only a placeholder for "not given" in the signature. While technically supported, we don't claim to support None explicitly in such situations (also reflected by the fact that we don't mention it in the type specification).